### ISSUE TITLE
Rotations causing GIFs with transparent backgrounds to turn black

### ISSUE THREAD

**Issue by claviska** [claviska/SimpleImage#159]
Rotations with angles that ARE NOT multiples of 90 cause GIFs to lose their transparency and receive a black backgrounds.

To reproduce:

```php
  $image = new \claviska\SimpleImage();
  $image->fromDataUri('data:image/gif;base64,R0lGODlhZgDIAOcAAAoFDxmQw4kDB6SIdMfGLIZLJgRCi/SKDMZLR0kQC0tMT4lpWNGMbbfG0OPl5MkCEM+njhAiLI+Ni0kmFOXGsQplmKhnPZClusRtK6dINllkbPGofJAnJ6puVOHW0CgJBgRKoMZrZVgqYNAlNWcrFVyEdAQoX66qqIpUNNaXdtC5sKtyR9K4D/z107ADDUhHki5GnGkHEOkCFs+GVupncTQoZNuUX7uLhKdbKWlHM/vJpSdopMDW5LdoLlex1Lm1sQRSrAQJV2hoaKknMfvoxQQ0cO6orvvYuEk2LykVEOebb1GCt8p6WGtWTGk3G5hYMN2JS9Kpq6l3XAQVN8rGwcp4P6yeJJAXGClKdedLWksaCqhaTeDKye/r6wVjz+0oOu4WKZCYod+5s6gbGwQ6hHSKqHY6HJZMJva3kXlHKwRXvIZ7d1cqEtMXJbdpRrp3WhOr39/c39Oam6+5xcp3fGkbJzEZP8vLzt59NpcDCryaFEZun8Gom7CZjWd9mao5NPfmBnSilDcjHESFxDejzIcrVCl51L6EZnyORHSx1e/QCaC1yOxeHHQzdEdadraiGMw+FCc6WSlXh8ZZSRR7p4c7GeR+VIODhVhZWE92bHh1dFGVxxRejCdanew3SDR0qgYcR/W8oueHjkt4repYZiQuTGJeZgZz4A82SeyqDBabzs7QLFg6gH2WsZI1QDWx44yWRK5IVqgaRNI3RSg4fCS67KCuSC6TyEwWPOmXWRcFBhhnqaKlpzlqmbc4NNhZZJo3LL+Pcvj39+yXneh5glcLGFpHQJllSTcLBthoc3dqZlo1IHdYSDZKXr+bijh6xDhYgXSWfHoZPl+VybytE/THxxhFf9xIVtkDFeGlhJ6MhbdrVxVMmXkDCbVXOBg5YzkZDRgWLbpYSQRq5Jl3ZERSnCRWxNzGELYmLJ2bmFt6oGeRd7lGO7l1RvkCGtLV1RhaomGLud+nqbgVIRpYvr28vNd4glhylDcrNLcXPjw+jNWJjwcjS8lmSFNngwAAACH5BAEAAP8ALAAAAABmAMgAAAj+AP8JHEiwoMGDCBMqXMiwocOHECNKnEixosWLGDMyjCNH3jBRouyJ2idPHhcHGlOqJBjHwy8a176AASPDnTswN8HMeknn5MqfFbmIyoLTptGjRmXIAHPNnhgPQKM2dCDKU1GkWI8qxdYGwb44UsMa5JKMZtazWmVg4/pLBVixYSlcQ0s37VquybjABepAjKeadesqVYvtwYMRcqDuTSkmC1ZPNIyEqmZkGKmrWWsSXmuYjoouizF6oIHUkxFhqFOjrkYK7eC7hQ0jkIcyNMU4w66CEdVFtTAaxFQTwwk47VbYD2bJ0WtbIgVPR4cJG3Yataib1VSTxqr5+F3DI0L+MG/ukGpxUcKOyJQeqigp1dW+cB/snfODeT3JO/Rr1JN1m2DQIN920qXmWFLudKfWZoXFRkdt+iUUhyjFVWeVfDZl0QV0X2QnTBfEAKZZgvTBZt8DLkwyXoQGeTAXgEcIU407nlRDGhjVdDFMhl30GGKCQCpF4oImGoaieCweJE9xX/RGjDvvdUEDKb3FZxMxDjiQjJDdDVnfA7EZ5oI4KiRJkAcU9tcbdOj16GYXFC4lRhyzYEOfl1sxGCaKD4gThZkCcUEKk0d0gdMwb2bpABdfYPOLB1E8MBiJeJr4nZEuzBOCGIAyWhyORiRoRI+KxmHqPr9Q8U4s9HWZp57+J4qJn2IsioENUsQYMRNtDpja0jt33EEFF3SYhWCrll6K6X10/MniksXJOcwvc8bBhRjy2EMMKVlcU+ed0b4GK5jkYoqOHEnuMylgX8jjgRj70OAJhkiJ2Oq9yTZo5L6+rNgcDXYmVRMYMnHnWon1Ibfnvi48gIA9Ufi715ZCalWYCxi7kEceGnOcsZ3hwlpkuftiOs8s+0gcFsUJXizAyzDHDHM3AtDcDc0cS8ogmEUuXDLDs4WWzFoYx5yHAEcnjbTMNzfdNNIuIGcYZz7/XLIv8sAVhxgIuCDz0UsrDTbSV5Rt9hUxxFBMAjHcLEDDVZfc8NxWo8OpVBwBA/P+xmIvvfQVYwzhy+C+/FH4H2Zo8QEAuuiCTDFNY7yvfXQ3bPXcSALlwSFXJO0132G/nMcQhvvCzumos/MHCR80/sHiACRgdtseW45ixrhLbrth82T9kwdSIM3xxh2PLcAV6JjuCwLMM+8NAt5kUEnrurzeOAAAaAEMB9u7XXvuGd/OsGGTvJUSFzdcofH6fBs/BjqnNz/J/NBDn0ESjF+PPfa6VPJHBv/bnujAR0C6GcZuK1GBK3A3vI29LAbI+EAd/sAO5s3vgvN7nhP4pwtwTMAJEwCHLgBAguh544QZyIDeaCY8AoovfOJQSRxuwD72OTAGsMseOy7Yj35Moof+PfQG/nTBBhRYoANu2EYHnrAMQWzBAltwwxZQeAYtICMGwmsh+PiUKZWoIBbgA103kBHBEQJgAt4Aohox0I9tAOADaUBiEreRxCTiAIl03AYUt5ABJ1TviqL73hbHoJJtDAFjDeTYy8bYODPqogA9rIIlZsCAXChBCcf4wDEYMINDMMGTb2DCG7ZByg5sox3beEMHOrCFNLTudchgoRbB5wuV3OAPYUxaDEZYPdhNYAai2IAw0UBMNHRgAdlIQTbEgC0IpOCZzwTlG0apRAug4HUfQEYCrsjCKwwBAbE4JMbmga6UxIEOGeObxpAWAwCAAwVJREEIb0BMHdizGvb+zEYo0ECBD3WBCB6ohhjQIMxsJDMYh5gmKp+AzQQ49HEC+MMwdHAEe9JhDC4Ywt004gAViBORoNMCOKKYyk8eIxhosCdFK2rPUOhANV0w1RHw2VJ5bCAFDEjoG1AQQW1q4aEZcOk+0eDSbHCADj/hQRTAyMCN1aEAdBTlDPahzJQe4apXVWlvUMODTdziFhfgwkxVigaDZiOn19RmArTwUxKIQQdogAADgpGCDRD0DkCJgwpuIA50NFAAwBDHNpjAhBtQwANHCIVLsYpVe6YmDq+AAxxUoYpNUCCrOlBsWc+ahmKw9bMJeAJcg7ECCxzxEPo030/iQAUV7IMOHYj+RQZi4QYGMCAFOeoCDxqggpUy1rGoSUQtAvANVFQgAHOoKEs1e1YzkIAEn/3pIdCQgg5Y4A2HSCgDskGBvWSJC85QwQwMOicHJCIAcPDBZY9AhPZmNTWvqEUpyhCGKVAiESxVqWbbUYnnRjcBb8jGG45xiNumYAZMsEE2ILQXKsiBqEdwQANUAQdKEOIO7G0vEd4rDAe8QhXhaIYjAIAKQlyVpnAlqAX6WwctFIMNa20HA56Agi0cQpnP3K6zFuMAl+rAA11IhCqK4IdmpCPDGr4qahww3HDsLwIBuKx+h9qOFv80AWzQQh2O0YFKpOEMHUCtMJWQgjLZBp9XjYP+DygRgQtgghlXJUILWrBhHbSgwx+ewhSCEIQp3IIKyoVrim1ghm1mkwRYPkMBnruFN8ygrsIsc3MQewQPmHcaA+CDlFXK3ooSATWRjQCfgwBlQOc3xRuwQQG0QMTPosC5ZuhACoYhzEhvdDFxYGwc2kvnJNszpRQNLhwqAIAgAMAAUSaCDgbgjPwSVAkWYAN0sfxqMxRgH8Uk5k0h0BwHxHnO4Ab3hllKzFD0xgEBoAQIUGGACmxiwwNIwgQ2TdANKOEMLU4ACVBQCTNswa5DjbTKwuJtOYc73O6tKEFTipoGPKMCFaBEPN5xBG0kYYRSIGu9q/BTZJgBBWdwBRP+gK1tJchBtXDpggM0fHBxjzvFSkDDpzu8iBNQgQgUaAI4WgeAHGh8zFtAxjufwEobkJygKbi1d5HMa4QnHOb6jHN7KUAOQbwSe+D4dcCVkIt2PJHLTAB4yVNAq6VLnddyTjK5CZoNBa8UAk0QRC95DoAkZOPX9b6kDZgAxQ7YwK7F3AAUPmObLmBVw4gfd1a1ncwUvGEBTRhiNi/Oyw6mNPAbyIUNOKnHNzBgmPZmQBQY7F3fxpmxiyemQSFggxTgg3GwTMLVG8kMomrb3jYQBRQIi10lZOOSSmBCYsjjAJWaPr/7FEM2IND4ueIv9tTbHxFTMNRn28AGCBZlKG3+ywA6HCIYcSC9dylAVKG2VLHKNyiZ6RqMYHRghB8Ahwgd2Tg20HPrm89+O0KAgydsIZU3wAfvwCJ8kA1llW3LtHrP1H7t10lIgD3UQ38TwAwMkG32lgKbR1iDtQ2ftQB38A7iZxuYIAUpAAHM50zOtIDfl10syAxmxDj5wz9pkHR2tQG/h4EMoIHbgAFbsE3bpAx4lST4MAFSUGDBMFcIdQhS8AZSIAUd0ISHwFD7M4X8o0l1ZVY2oAS2pYGo9AaVQEZkpAUnYCYKoAtIcAxNyIQr0AHkcAxuuABc9gYuSIVT2DgTcAzMd4NamH2ktA394AZ1kE1gqA1mcgL4Aw7+zIACC4ACiciIjOiGx4AEL0iH2IMMTvAG0ISDM2APTBAC7dBG/XAGZLRNyCCGgPIDObA4grAMOZAGzJAGKMCIaWAM+sNB1vM6E5ADKHBjmkgHTNAOb+AGIdAPVdAPEwCGZMQMQZgk70IO0pcEgiAI+CAIF0eJb8QGZpAGTpAGaVAAYWZbh4BgqOQGQDSMZgCGr+MEPwAo/+ABFCAGD0iFk0iHuuAEBVAAKMBlHdAO2cWHdESO5NhDokhGH5AAHsiO/+AAFBAKKkCLjUSHEfgBH9SNx8AE4OhJoPSP4iAOapQBYLgMynACIcgiXMBd2SAF3JgGcrc/E5CNTmAG21j+AMeAXdwHjtOkRxvpBj/kQ8DQU8xQD2WHkH1hgBAQDG9gAU+QBhPABk7wkt1YAEQXSp9kW5sHSm7gBuKAlUA0CeeYTR+wBkGJkO1ogo2XUO1gWk+AA6ZlAcCISsBISdwnSlfJkR1pRdikjiPJjh4gV0eIUDPwBu1gXablBmtpAalUYOCogWr0h07AcyOkAHOAcmI5EB5QD0aJXdM0Taallqb1RO3ABDPQSdnXD51YBYRVAOHQZ3oGCriAC4VwAxAwLJNZEC3hDCtwm22pR/13Bpz5mZ9EWKbJBHhgCfbgBvhgbHo2BXbQDd8zAuJQXrM5EA7wA8GwltbVAf13DGr+6Qbt0J1VEJyWQJwhkANOtpqs+TZwcxxtkAzyYGbR2Y71cAxPUJj0aQFXyZ14kAyWwAQoEA7JCQoAWgw5EzBp0Qa/4C7ReQeLsAdFgAQFIA4WgAM48EScaQE98J3huQWvZ54BOqB0IQNt4BRKxyJ6pQ7cUASg8AECAD9ZuZY90AMY0A5VkAwIwAb+OQUAiqP8kAAccyuBkSCzkAwjGhoO8C6kEAsvwA1kMAXFEDUoMg9DcDrecJ+T4AtXkAD/CaCgwA92wDE/ihTYMAuiEJZwQQGicBn5AANqYACgEAMgoxYg1T41kwRZuqX8EA55QKBfmhQGKgegAReLIgoYIgv+5rCmoNANI6IZ2JBODmQHWmoC/GACJlAEJiAAPrqn3IEYkqkSfUEM9DIEhQoC/CAAxWEcF+MCpTCpRbCqZNCqZGAHl4qp9aIW9jBwF+EBRjAcRyEL9OAFIGACbvqhaiELNeCqBmAAIJCsQFAEeSCrB+Mo7pkRcSAPgooU+WAO46AGZIALD2CqbSALsiANIkALZHCsyXquQAAEIGAHpeqssxqm+5CXEMEFw0AvRvEF5ZCtIFAEtNAI/toIrPACL6APMMANygoCQMAJmbCwnAAEavCwasCs81ExmHocvjCkEBEHRnAgRgEGjJAKimALXgAEZKAGXnCyJ/uwXgCxEAv+D+uwCoAQs6tQAg6rBkAAq9zxphT7o4SRHBjbELhhrzdxADELCASwCwagBuMwDijbtChrsmqwDkUbs4pwDjSrrdIwH5YyKYGxFeTzswrhAbqqFXgwtYGAsEubtkvrtCm7CwQwtTF7DlYAD2pgAnlQqq8xNbCxs2dxHJ0hrwcRB2NLItgABopQtKuwAyDgBWrLtI7Ltl4As1OrCCzwCL3wq1kbMnwyOWvBtzkbG+ywYw6RGzbRs9gACYcbs+sAAkrbuGnLuGxrC0VbtSxQuZxAqbTQrfiSO0ayt30rAyZTTg7hAfSiFmICCSxwDuewDjbruo3LuEyLsr1ADbVbu9T+CwvcYAIGAATSwCUts6jEs0Vg0q6le6oYsw0QYQ/24rVjYAWPQLNJm7anMA7z67qwy7pesAuI4L6PQA2IAA8GYALJmrvIsqg1szfho6cWczuIFAJkGrizQL5K0QjwAA8gEL9LW7/0a7/Rm6wrywmU0AsV8LCrqq5AIAKduxkY0zbFADk9Sr7la74bEwt84BDy0AZ9ywraCgJqW78arMFri7Lq+rDpesFkoL1qkKwGIA0jczRjZAJ4qsB2cTvtEwvRqhDJ0AZ8qxb6gLbyu8H0O79irLawO7JksKqq2qoOq64gQAbSwDOx4TUxEA4mwMSe+70M3D7b8MAG8QtSnCD+sgC9zvvDQPy4T5uuLAuxRWwAZFAHU3M7ApAA/MCtd+y1ecw3b/AQ9qDF0ZIgL+C6Y7zBPvy8jxu9TcuySszINZAzGNMNoGAHYAIk9UI5iLQ0HLAPD6ECnBwkMpAPgpzBwDzGhby2puy0UKvIStyqRYALLBwOXQoy72q+68Q34sDHBuEAI1AxmtEIgizM8jvKHFzGp7yyRKyujLyqlAqpoGDHWismGSM6SMMBotsQv7AuamEOv3wKwjzMZBy9xZyy5FzOydqqkooFl6ANsQDNAkPL07w3hzCAECEP0KwU84CtznvRQAy7/3zKqGzO53wPLbEPbRCrhCvNDhQzW7D+qQvBBRFMH4Xwyxf9xa9LzOIM0CzrsOfaqtbQAB8CMCHDGYw6MzXjClQwEQ5gDyAjpq1wsjHd1K9b0ymbyEU80GQADXHwITccLkANUkItABxQwxTBBW3QBjRQKJsA004901D9tAGdxGzcxmeMBVdtHTTxGjIMNm7j1c5gEfDCG7r1DKWc1ho92BxNxG6NsOd6zkUwB6ohD1+wGXm8SE5zBX2wjhWRJR0WB3dgCEoLvUw92P4MuSbb1jddxG9N0N/ACw4AH0NzyZLtNlfAByr9ECrnAO9QD7ywA6O9sgANub5tzORs2ueazGfsCHfQAHONGl0gB7PA1TXjNF892w/+Ydt3UA/psAifgLI0vdGfDdXHbLM4/dapbACU2go8oNkgqNxdAF5/YDNN0zauINvSegc/wAt30AXPsNu/7dtQS9rgLd6p7KrfEJkqRwWqEn6m4gHg5Qp10DY3UwhUIN0Q4QC4/Q4fMgj6/dsZ/rSlzcYAjqyuWgTQcAdx4Cb0XQ/BkuLC0gcLoI1psAZFzVHBkhoYzrJs3d/fnciKLNzDfazHqswmEA/3/SZ3wAu8cAI/8AP1UA9UUA8nEAaXEOUn8KcZ4QBb1QXxULM6rrJbruPpasLDfcEgzsjKXAStEH5b9SHvkA4SIAHpwAtvfgJwng7akA5TnhKrnRrTYLP+Xd7n5SzcH+7jP37GJvANF1Diyu1PcXACl6AJUd7ml9Dmbh7jGaFyqdEFi2DCX67lX87na7zpiB3qYi7ornrGq0oLi1DiPZLoMdUA6aAJQiAEyqAJji4BvCDhE57nHTYHCAvmPD7VYB7miS3oP07mZ0zo1jAHVu5P/lTb9B0GawDrpiAEayABy4gRWeImvdIJw+3hwv7tPU7sxh7ihF4EWIDcVv4mH5Il7/AO9d3otB7p134R2U4q76AO5vrtyHrBPT7ejDzupW7qpl7QDZDuvXHwWRIHwOLkbL4GUc4L9ZASKkcqccADvEDs5TroxprxGB/wpV4Ex47O6YwF9aD+KG9S2+1OBfUdBhLg8JcA8YA73aWy8JJgrBgP8B5P7iAv8iI/qZLKD7SA7stOKrb9Dk1+Amx+CWuw9Ej+DlRuEYrSK8BCBWEA8jl/7FivzKbO81zv82gsqaAw4OFX74rCA8dd320e7ZqwBrzwA1QQ81OB4O0uLD8ADej88azK9XqvqmhMqeksqYBf6Ivg9Nlu5Q5g9ioP5y1P69SeDkp+B3CvEOzuAe+g8GffCt+g932/+arq9Zz/954P+Lce9SqX8j+A9BIA79OuCWHA5OhuET2iKu0+9XJ+D15/+3+fzrof+Lzv857f961Q+VF/3onP8o0e6wqACUIgAW5PBST+ftmoYfSzf/Rs7gi9f/3Yn/18j/dlQOJjr/BUMAdsnvqwLgQKcP6YoAk21+QgSBF57u7OLyxODu0asKqgEPj8kP/5b6f5r/2Az6oAQUagAYFF1N15FychDyq8wkiQcEmTEE0SNClQYEwILyp36tVz8E/kSJIlS3bpIuzdiXp3Ov7gZVEDwSKgTIACxQ/nziA6d+68idOEiSICBxpAipQMNIQJ39WLCXGNpolCTtz5gUkBpnT1GlD5QcXk2LEOUHaJMwch2HSXhGiwRnPozilTgtSt+zOozaFFySAFYQBE4MBkrC26oxDrQ4tCHAu59OPdOyqYTEng9UPzj5BkPf/+cyBMtLA4HaGGWSPElCSlZIoExYv3bmygJo4GFiyYMMEyVN7duXPiBK/UmDBBBlk6TMeYvJzz6vyZrFnRXSjDtGhKiKS/rV/jjA2g7uwpfP/uHpw+PdJOc3xPpizB8fETHhT+0JSudESIYqV7diBAB35rACa3tJspKZpe0ym8KcTLyzbAdFOvQhDIuMe3OBT7QT6KqFConnQo+qitruL477MBC4QJtYmUWcOPwRQsaC67xnuwLqLIIMxCEIAAQj2kWkFIoTgGTIel63i5RDvnfvMguhSnC+6hdNLB7IQG5tglt6RqvKkuAMa8ccceBwMyzTTXw+IC4KIUMCW0OhT+4rg1JJBsyhQH/KGehAIU6Z1P0PsLTPDEFA+UIgBLL0g11VwPGo4mM0vOLhyoRxPtqCKxCz0/LcmBXmbczagivgMlCPESLerMR1+NNLOOBJwMK000UEYTyzQ5QUpQP41nvVJdew22B++yTT01XoV1MDKwKIOXeqigtp4fTpAAwa1M0YQXFH/9tYwKaSR2LlByBMA2HtFUc1lm0yPDBEcuSSeMeiU44SFdjaOIM3B/VSdIUr8s19zYFmWX2Xed5YefSJppRgNAN3zqhD59/TfFReDxsdCiij0UgCCS/RFId9VAGeU1l50x3p0imSNjmUt6ZwcLBTP1VIPzAizNlIH+OFnlNWdkkB8TwvB05pkdsPlmgogt1qe8imB32Z+vdndogoYyQZ1vlZZ5FEfHzZnrhs+dwgTBTLYaa5MhxU1nLO4Ae+ZgBRaY4FOjzmknHn1OOXCgBx+7ZWIvqFvmVgpf7+mCTfAJJ4QJD7zyt4Gc0YC9/cA48SkvGFtg3J7em2u91ga6ctXd/THzgiSh2/NfGxi6cceLhRxynfhZN/XVfya88cNln52ekoW8cEHcc9od4bZ/Fxpzw10r42viU2wg69ZHJ13nm763iQxHoQde6wsFkgTx66dsYIdHheye69xt4r1k31d/P3PHL0h6feneeUazcFO2uRjNJjBA0/P+BKc96Y1OIK14h//+E4dBqEx7DnycARsmLvvdL2vvE12hlhJBCapoCdFroHdew6Dv8aMVo0iP1QgHKcyJboCGmUPnSjiSaVgub+dzTbyGYsChlOECjepgDVunRP090Ho7LEkPK7c9veHOgKC4RwM4lsQkrsmGN4RGK6BIlh56YYHryRkL+VKOdyyhUQoLnRIJRgb1jTGKXsCj4IT3uCGC4htxOGLCaKjEL96wenYkiQMGMQ48MvJnNwRT7iBXBAfMwXhdBGGQ1EDF1nQidoj8xzsMMQ5G4tGMLJue94Y4lDk0YA/bgxXrvAg/pIQBlCJpwClISco8oqyJrpHfKi/+4AAOYhKOTjPAHkhoxzhsYpe8zKPoHAfMYJqgDP9ohavgWMPaEYYbi0CkAxZxCnKOQ5eOVNluSFdNE/jhH3cYlW646cXBXVBI8bDjOxJhCF2eE5qoxKAquQaNAMVjXYIE3O9q14kGSDBAVODCOM1JynP2EpVVrCYWIjgHblAIBEELnBlFKrgaipF4HhADBGwwgyq0wwKm2MEzdxlNYXUPd0PBAjgbEM83htSUP7XoJoG0h08qzQEotQEU8ICHKmDAqd546TNNyTr9KW+FQ/kGONtYIbah7Kcz/ekjv6lDUDlADEmFQlrTylSn9qAHONDALoIqvdyYiih3JYr65qD+nvF5tZEyheYp0zSKBvTvX2aFQi4Uq1i1KrWpbX0CNIBXoe6ksS9FuOY/5tCJnqrBlIAFrGBndBCy/scBFEjrYlW7VqY+9q1NgAdVB1Y2VXIONDCsml+9AFqwihYpvYjZvzzAACjYQLWrTWsVXPvWMygATQP70seIUiyiiiSb9vMrb/2JTiAYwBrfoMU3bPsrB0CgCsVV7AbUu4HFJpetGHgrDs6QBlroRkG3K91rsNDQfywCb7rd7TO3a8ZOOEIZx3iCBeJrAQiogAulNQkXWJpU4+ZivezNBVqVWwW34sDDZyiAE0rBoy/h13tFsKVI4AG4zwL2nDvwBzm2sWH+5WJguW8Ihn9SpIJ2VGEG6FWChder2JWytB0d/vAZKmGGZZSiKDjDL16xMBI3muyzf6XoKfawhh7jYQZLVepSl7rcFagAwv/oAh967OMZ2MAGQbZwLpSQ1Bn02K3xlW8BlGwGEkzADkYzSkFUmdl/DMpqQN2lLu8hhS//GA9pRetav1zjt/bAGTomSxyc0YMVtGMGbXazEoLsZigY+cjxfcIZQFyJAizZDE5gwwTCEQ760SUcdpCASPzru6/y0h9v8LKX1ZqLxBY7uSxtaoefEIxljsUDA8BBD9rR40+3mQE2YMCnp31kDz/B2yAuQAHMMO5X89kJfdZCEpLwgQ/+IAMZ4GCGSKYxuJHO1BEd+HFjj8tYtCKb0h92Blk9IIVoW2Dbh2CztqdtAQvgwNveDneryT1uJ1T83OdmQ8a1MIGNN0Ek8fApKdUAjWOc99Gp3TdyP63cU7/V23zQ4bOfgAMLdHrbN2/HChj+8CdEPNxmSAPFnWBuEpx7GRhfRqzZsAaRcBRopgQCFprQZbVWOOVEprOR4atgh5/hGCqYzgAKMHOG17zTK0B7wx0OcZ8HvdwWh7UTkj73ubNhGctIh0j4YIxv8IgMpWiCBZRbakgfl73rHTWdWe5SZeOgAAPwQFn6EG5vl93yOEDwE1Dg8wIE3e1wJ0HSMz76CWT+fAJIWEYTYueM+S4DCSiwgI2Ve3I3L1YJFz58hul8iIX3gOFrPwMK+ECWH0T84TRP8MxRoPmIp8H5QH813O9u+gkIYgLgqP4EtI8E/YiE9R6Odg+cWgU8MCHf6I2zeoUsZzevfNtubXiqe96HJ4okHSgAegGWz3MU9L/z4na7t5M+0rM+cDBAAxQE61OGyBOJPqgEVfMwt8KANWMzNzOu2xOyDUi8bKuzIys7h8OBzSsAcmg2kjiBoUuDiNs8Eew8chu6F7S4uxO96hOEA3y3A5wAJ9AGkvCANHjAMwC/O6O2TyuuC1QvUVOCFGi/GeC9nPu9hxNBFCjBkXCAPmD+A4tzviwUQLiTO7kTvVizPkHQAmT4gCQwwNNzghxIgx9IpAUYOlYDQpqTtiH8tNqTM1GzQA6cNp1rOAuAwojDNJpBAY5jg6IzRFgrxIwjgdGjvuzTPuw7PRnswhwgh3owCSpYBi1gAzPQMxB0q2lLODdTwjxkwg50qYbLPG8TwTX0DE1bgTS4Pi0ABy2gxVrcuNJ7RDHEvitsAm0whgmQwRwQRnJIhzwxiSqcxU1kNW+Tw22rthlgAAYIhlJUOD5cO83rv81zPjb0DC5whpZCgRxgA1lMAjJsNwOUxdK7QjVkBm0Qiw4ZgCZghj4ACelwACoYRGTYxHA7A7IzOGf+PIRnnIE3eIM9LDsozMZwywEU4MZWVIFg4D2GO4b+S0HnaztmWIABOIEpBBUqyLk0KMRxo7wEqzmc2zabSzsEy8aV1MaFDMSy8AAIcAYpOAQpkIIVuElyWAFyIIdj0AZt6IN3OLMU8QBncAOXCkkV9Ee0Y8qmtIBUxMaVLABmcL4cWICXlA4u+AEVoAwqUIETgLn18cpD6AEUuEKR1L+ZO4aySzuJ5D/9iziqTEE1zIEB4Mhbko44KMoViLuhMz6yOwbMS8WEDLeKzEJmoMt5HEq8PEYuqAdxrLj867+HyzzCzEK6TAPMpMurZMyMqcJFLDrJZEmKrMqgywHo08zLHGAGZvCWzsyYgYPFK0TBFCxMz4vM21yGHHACY8gB3lTNhnRNcCmNdsABJyi9o+PCipNNupPBu8uB3EyDXglOmSkvKXApM4g17dO+7OROXES978xN/bPE6ZQZLqACZ9i2YwDJ6wOHGzzABLQ+u0vDzsvIuyRPPekCD+ADCOC9lnKpJ6DK00xDM1DDLEwDFGAGjay/+/yXOLgDCIAAiGTKmvvAtTwGKRiAPmCJI2FQ2fGAd/AAFRBRmXSGPlCBrZyWD13QDmXRFmXQgAAAOw==');
  $image->rotate(45)->toScreen();
```

---

Original image:

![test-2](https://cloud.githubusercontent.com/assets/55639/21818121/9f6cfbfe-d734-11e6-867e-22e72a804b87.gif)

Output:

![test](https://cloud.githubusercontent.com/assets/55639/21818128/a4a24ac0-d734-11e6-9b46-75ebfdeae59d.gif)

**Comment by peter279k** [comment]
Consider the following code:

```
$gif = imagecreatefromgif('./mario.gif');

$rotate = imagerotate($gif, -45, imagecolorallocatealpha($gif, 0, 0, 0, 127));
imagecolortransparent($rotate, imagecolorallocatealpha($rotate, 0, 0, 0, 127));
imagealphablending($rotate, false);
imagesavealpha($rotate, true);

imagegif($rotate);

```

It will fix this issue :+1: and we can add this code to this ```SimpleImage``` package source code :100: .

**Comment by claviska** [comment]
Looks like this should have been closed with #222
